### PR TITLE
Add ability to read in access token for quay private repo authentication

### PR DESCRIPTION
**Pull request by clacroix12**
Jira: [OCSQE-237](https://projects.engineering.redhat.com/browse/OCSQE-237)

Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

File: ocs_ci/utility/utils.py [hunk 1]
```diff
@@ -1426,7 +1422,56 @@ def get_next_version_available_for_upgrade(current_tag):
...
+
+def get_ocs_olm_operator_tags(limit=100):
+    """
+    Query the OCS OLM Operator repo and retrieve a list of tags.
+
+    Args:
+        limit: the number of tags to limit the request to
+
+    Raises:
+        requests.RequestException: if the response return code is not ok
```
Comment (dahorak) [inline ocs_ci/utility/utils.py]: `FileNotFoundError` and `KeyError` exceptions should be also listed in this doc string.

File: ocs_ci/utility/utils.py [hunk 2]
```diff
@@ -1426,7 +1422,56 @@ def get_next_version_available_for_upgrade(current_tag):
...
+
+    Returns:
+        requests.request: response of the request
```
Comment (dahorak) [inline ocs_ci/utility/utils.py]: Wouldn't be better to return directly the list of tags, instead of the whole response object? (like:
```python
return resp.json()['tags']
```
... with possibility to better handle any other exception there)?

File: ocs_ci/utility/utils.py (hunk 1 above)
Comment (clacroix12) [reply ocs_ci/utility/utils.py]: Done

File: ocs_ci/utility/utils.py (hunk 2 above)
Comment (clacroix12) [reply ocs_ci/utility/utils.py]: I originally didn't want to change any of the logic and just wanted to cut down on duplication but this is a very small change and makes sense. Done!

File: docs/getting_started.md [hunk 3]
```diff
@@ -122,6 +135,28 @@ If you don't want to use the shared key, you can change this value to
...
+Create a file under `ocs-ci/data/` named `auth.yaml`.
+
+#### Quay
+
+To authenticate with quay you will need to have an access token. You can
+generate one yourself by following [the API doc](https://docs.quay.io/api/) or
+you may use the one QE has generated already. Ask people on ocs-qe mailing list
+or chat room if you don't know where to find the access token.
+
+To enable ocs-ci to use this token, add the following to your `auth.yaml`:
```
Comment (petr-balogh) [inline docs/getting_started.md]: Can you please share the token on some of mojo page like we have for pull secret? And add link here?  You can do in different PR if you would like to merge this as you have approvals.

Comment (clacroix12) [reply docs/getting_started.md]: @petr-balogh We have the `auth.yaml` in our ecosystem private files directory as well as the token being documented in our passwords file with the other quay information. I can also put this in a mojo page if it will help users, but I was trying to avoid putting it in a publicly available place as for now it's legitimately _my_ token.
